src/runtime.spec.ts

```diff
@@ -77,4 +78,21 @@ describe('Runtime', function() {
     expect(runtime).to.be.ok;
     expect(Object.keys(runtime.keyProvider.keys).length).to.eq(5);
   });
+
+  it('should NOT create a new and valid runtime with only passing mnemonic and empty account map', async () => {
+    const runtimePromise = createDefaultRuntime(web3, dfs, {
+      mnemonic: 'annual lyrics orbit slight object space jeans ethics broccoli umbrella entry couch',
+      accountMap: { }
+    });
+    await expect(runtimePromise).to.be.ok;
```

Comment:
```suggestion
```

`createDefaultRuntime` will return everytime a promise. A check for truthy is not necessary at this point